static/bidder-info/dxkulture.yaml

```diff
@@ -1,6 +1,6 @@
-endpoint: "https://ads.kulture.media/pbs"
+endpoint: "https://ads.dxkulture.com/pbs"
 maintainer:
-  email: "devops@kulture.media"
+  email: "devops@dxkulture.com"
```

Comment:
@kmdevops  prebid team has sent email to verify above address. Requesting to respond with "received" message on email thread